t/cobrand/land_type_for_problem.t

```diff
@@ -0,0 +1,212 @@
+use FixMyStreet::TestMech;
```

Comment:
Could this all be part of `peterborough.t`? I know it's got the Bexley stuff in to check `land_type` doesn't get set on non-Peterborough cobrands but that's OK I think.